db/migrate/20220318023014_create_members.rb

```diff
@@ -0,0 +1,14 @@
+class CreateMembers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :members do |t|
+      t.string :name, null: false
+      t.string :facebookUrl, null: true
+      t.string :instagramUrl, null: true
+      t.string :linkedinUrl, null: true
+      t.string :description, null: true
```

Comment:
Please check the description of the ticket because `NULLABLE`(allows null values) is not the same as `NOT NULL`(doesn't allow null values).

Reply:
updated